lib/MySQL/Diff.pm

```diff
@@ -179,7 +179,7 @@ sub _diff_banner {
     my $now = scalar localtime();
     return <<EOF;
 ## mysqldiff $VERSION
-## 
```

Comment:
Sorry for this - WebStorm just made that automatically (even after I reverted :S)